app/src/main/java/com/everest/userinfo/model/UserViewModel.kt

```diff
@@ -0,0 +1,54 @@
+package com.everest.userinfo.model
+
+import android.content.Context
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.everest.userinfo.HomeActivity
+import com.everest.userinfo.validations.ValidateFields
+import java.security.AccessControlContext
+
+class UserViewModel() : ViewModel() {
+
+
+    private var _user: MutableLiveData<User> = MutableLiveData()
+
+
+    val user: LiveData<User>
+        get() = _user
+
+
+    fun setUserDetails(user: User) {
+        this._user.value = user
+    }
+
+    fun validateInputs(context: Context): Boolean {
+        val validate = ValidateFields(context)
+        return user.value?.phoneNumber?.let {
+            user.value?.pinCode?.let { it1 ->
```

Comment:
instead of using it1 you can can name it as pinCode only.
because in the code below it is not easily readable